specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/definitions/Tasks.json

```diff
@@ -635,7 +635,7 @@
       ]
     },
     "MigratePostgreSqlAzureDbForPostgreSqlSyncTaskProperties": {
-      "x-ms-discriminator-value": "Migrate.PostgreSql.AzureDbForPostgreSql.Sync",
+      "x-ms-discriminator-value": "Migrate.PostgreSql.AzureDbForPostgreSql.SyncV2",
```

Comment:
Will this child type change cause breaking change?

Reply:
We are making the back-end backward compatible. Same inputs and outputs will continue to work with the 'v2' type task.